docs/products/kafka/kafka-connect/howto/gcp-pubsub-source.md

```diff
@@ -212,6 +207,5 @@ replacing the placeholders for `GCP_PROJECT_NAME` and `GCP_SERVICE_KEY`:
 }
 ```
 
-The Apache Kafka topic format will be the default bytes by default, you
-can use the AVRO schema by including the `value.converter` and
-`key.converter` properties defined previously.
+The default Apache Kafka topic format is bytes. To use the AVRO schema, include
```

Comment:
Should "Avro" be all caps? I noticed it wasn't written that way elsewhere, but not sure which is correct. 
```suggestion
The default Apache Kafka topic format is bytes. To use the Avro schema, include
```

Reply:
Thanks! It doesn't need to be in caps. Updated it.